src/collection-view.js

```diff
@@ -643,14 +653,37 @@ const CollectionView = Backbone.View.extend({
       this.render();
     }
 
-    const hasIndex = (typeof index !== 'undefined');
+    //const -> let, because value may be changed later
+    let hasIndex = (typeof index !== 'undefined');
+
+    //if we added atleast one indexed view _addedViews perf should be skiped
```

Comment:
I don't believe that this is working. if 2 views are added `preventRender` without an index, only the first view will be attached. 
Additionally if someone adds a view with` preventRender` without an index and then a collection updates, the views added by the collection will not be attached and only the `_addedViews` will be attached.

I think the best approach is that `preventRender` should also prevent the `_addedViews` perf as the next render is not known.